panoptisong

```diff
@@ -2,12 +2,33 @@
 
 # move to source directory
 cd "$(dirname "$0")" || exit
+cur_dir=$(pwd)
 ## load parameters
-source ./parameters
 
-global_attributes="-a experimenter=$experimenter -a room=$room -a species=$species -a lightson=$lightson -a lightsoff=$lightsoff -a jackserver=$servername"
+prefix="$1" #which set of parameters we are loading
```

Comment:
$1 is the optional second parameter, that allows you to use "profiles" of parameter/bird files. If it's not present, then we just use the standard birds and parameters files. I've documented that.